src/extensions/ignite/plugin-overrides.ts

```diff
@@ -0,0 +1,20 @@
+import { IgniteToolbox } from '../../types'
```

Comment:
Don't bother reviewing this file. No significant changes.

Reply:
For future reference, if you had used `git move` instead of renaming the files these changes would show up as a name change instead of a file create and file delete. Also, the files would retain their git history.